tigerpath/majors_and_certificates/majors/COS-AB_2018.json

```diff
@@ -237,16 +237,25 @@
-      "no_req": null
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "<i>Editor's Note: COS-AB majors do not always enroll in a numbered course for their Independent Work, so TigerPath does not check this requirement.</i>\n\nAll A.B. concentrators engage in independent work supervised by a member of the department. A junior project normally involves the study and solution of specific problems, often associated with a research project. It may require a significant programming effort, a theoretical study involving the design and analysis of algorithms, or an applications problem in some other field. The results of these efforts must be presented in two written reports and posters that correspond to the work undertaken in each of the terms. The senior thesis may be a study in greater depth of one of the subjects considered in junior independent work, or it may deal with another aspect of computer science and its application.\n\nIndependent work is one of the unique features of a Princeton education. Independent projects typically arise either from an idea that excites you or from an idea put forth by a professor in the department. The department maintains a wiki of faculty research interests. This list is a good place to start in looking for a project or advisor. There is also some general advice on finding a topic, an adviser, and a successful outcome here.\n\nOnce you have a project, get a form (located on the bulletin board outside the Computer Science undergraduate office) and sign it along with your advisor to let us know what you'll be doing. Your project is then between you and your advisor, but there is always a COS Independent Work Coordinator who coordinates all independent work, and schedules public talks, checkpoints, and other requirements.\n\nIt is possible to do a project with a faculty member in another department if you have the approval of the Independent Work Coordinator.",
+      "course_list": [
+        "COS 397",
+        "COS 398"
+      ]
```

Comment:
COS 397 and COS 398 are actually optional to fulfill Independent Work requirements for COS-AB but I didnt' know about the optional track trick because when I made Independent Work null, the root wouldn't turn green.

Reply:
Could you link to a departmental source that says that this is the case? Because as far as I can tell, the department webpage, and the course listings themselves, both very clearly say "Open to BSE COS Concentrators Only".